uvloop/loop.pyx

```diff
@@ -1775,7 +1775,7 @@ cdef class Loop:
                         if reuse_address:
                             sock.setsockopt(uv.SOL_SOCKET, uv.SO_REUSEADDR, 1)
```

Comment:
Isn't the same problem here with `SO_REUSEADDR`?

Reply:
No; the place to look is `uvloop/includes/stdlib.pxi`

    cdef int has_IPV6_V6ONLY = hasattr(socket, 'IPV6_V6ONLY')
    cdef int IPV6_V6ONLY = getattr(socket, 'IPV6_V6ONLY', -1)
    cdef int has_SO_REUSEPORT = hasattr(socket, 'SO_REUSEPORT')
    cdef int SO_REUSEPORT = getattr(socket, 'SO_REUSEPORT', 0)

Note that there's a check to see if the variable exists, and then a python definition of the variable with a default, guaranteeing it's defined. This is how the code handles platforms that don't have the constants defined. For other constants (such as `SO_REUSEADDR`) there's neither a check nor a python definition of the constant.